app/views/users/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>single user information</h1>
+<section class="mt-5">
+  <div class="container">
+    <%= render partial: "user-header", locals: {user: @user} %>
+  <div class="row border d-block m-0">
+    <h5 class="m-2">bio</h5>
+    <p class="m-2"><%= @user.bio %></p>
+  </div>
+  <div class="row border d-block m-0 mt-3">
+    <% @user.recent_posts.each do |post| %>
+    <%= link_to user_post_path(user_id: @user.id, id: post.id) do %>
+    <%= render partial: "posts/post-preview", locals: {post: post} %>
+    <% end %>
+    <% end %>
+  </div>
+  <div class="row d-block m-0 mt-5 mb-5 d-flex justify-content-center">
+    <button type="button" class="btn btn-secondary">
+      <%= link_to "See all", user_posts_path(user_id: @user.id) %>
+    </button>
+  </div>
+</div>
+</section>
```

Comment:
- [ ] On the users show page, each post should have the number of comments and likes under it as seen in the wireframe. Kindly add it.

<img width="299" alt="chrome_CC9a1vca3z" src="https://user-images.githubusercontent.com/81468675/163545489-f8ee6960-c3f3-4906-88be-9fee87f776be.png">